Kitodo/rulesets/dfg-viewer.xml

```diff
@@ -44,7 +43,23 @@
                 - required attribute 'id':
                         Internal name of the division. This should be a
                         xsd:NMTOKEN (that is, do not include spaces).
-     
+
+                - optional attribute 'processTitle':
+                        Template for making process titles. Production offers
+                        the possibility to automatically form process titles
+                        according to a schema. This is a string of metadata
+                        keys and constants separated by plus signs. Constants
+                        are identified by enclosing them in single quotes. The
+                        special metadata value TSL_ATS is given by the system
+                        with the author title key or, if no author is
+                        specified, the title key (4/2/2/1).
+
+                        When templates are used to form process titles in a
+                        rule set, only those divisions that have a template for
+                        forming process titles are proposed as the top division
+                        types for new processes. Otherwise, all divisions are
```

Comment:
why is there a restriction to the top level divisions depending on the fact that they are defining a processTitle or not?

Reply:
Otherwise you cannot create a process title. That's what Sebastian had suggested in his email from 02.12.2019 12:34 o‘clock, and I think that's a good idea. That there were always all elements available was also a thorn in my side. In 2.x it is also the case that only the explicitly defined DocTypes are available for selection, for which a rule for the creation of the title has been defined.